environment/src/main/java/com/sequenceiq/environment/configuration/DatabaseConfig.java

```diff
@@ -3,8 +3,6 @@
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
```

Comment:
General: I can only see two `DatabaseConfig`s being updated here, for `environment` and `freeipa`. What about the rest of the services?

Reply:
thanks, nice catch, added the missing ones, please check it again :)